src/lib/evil/evil_main.c

```diff
@@ -28,16 +28,6 @@ evil_init(void)
    if (++_evil_init_count != 1)
      return _evil_init_count;
 
```

Comment:
Haha, you could have removed one more line

Reply:
True, true.